common/lib/mqtt-messages/src/lib.rs

```diff
@@ -75,3 +106,18 @@ impl<'a> TryFrom<RawCommandData<'a>> for Command {
         }
     }
 }
+
+// impl for ColorData to convert the Cow<u8> message.data()
+// from EspMqttMessage to ColorData(rgb)
+impl<'a> TryFrom<Cow<'a, [u8]>> for ColorData {
```

Comment:
Can we `impl` this on slice? as Cow derefs to slice? Then it would also work on slices and other things that deref to slice.